syncd/syncd.cpp

```diff
@@ -1269,7 +1269,7 @@ bool isVeryFirstRun()
 
 int main(int argc, char **argv)
 {
-    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_NOTICE);
+    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_DEBUG);
```

Comment:
Simplify this pattern?
   
    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_DEBUG);
    SWSS_LOG_ENTER();
    swss::Logger::getInstance().setMinPrio(swss::Logger::SWSS_NOTICE);